package.json

```diff
@@ -72,9 +72,7 @@
     "url": "http://github.com/jquery/globalize/issues"
   },
   "dependencies": {
-    "cldrjs": "0.4.3"
-  },
-  "peerDependencies": {
+    "cldrjs": "0.4.3",
```

Comment:
After moving `"cldr-data": ">=25"` to `devDependencies` and adding a copy to `peerDependencies`, the warning message no longer appears. Do you want me to update the PR?